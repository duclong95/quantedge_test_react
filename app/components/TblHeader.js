import React, { Component } from 'react';
import {
    Table,
    TableHeader,
    TableRow,
    TableHeaderColumn
} from 'material-ui/Table';

const TblHeader = () => (
    <Table>
        <TableHeader displaySelectAll={false} adjustForCheckbox={false}>
            <TableRow>
                <TableHeaderColumn style={{ width: '10%' }}>Code</TableHeaderColumn>
                <TableHeaderColumn style={{ width: '29%' }}>Company</TableHeaderColumn>
                <TableHeaderColumn style={{ width: '19%' }}>Price</TableHeaderColumn>
                <TableHeaderColumn style={{ width: '14%' }}>Value</TableHeaderColumn>
                <TableHeaderColumn style={{ width: '14%' }}>Change</TableHeaderColumn>
                <TableHeaderColumn style={{ width: '14%' }}>%Change</TableHeaderColumn>
            </TableRow>
        </TableHeader>
    </Table>
)

module.exports = TblHeader;