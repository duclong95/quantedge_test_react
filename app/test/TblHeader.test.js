import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import TblHeader from '../components/TblHeader'

test('Render Header',()=>{
    const header = shallow(<TblHeader/>);
    expect(toJson(header)).toMatchSnapshot();
})