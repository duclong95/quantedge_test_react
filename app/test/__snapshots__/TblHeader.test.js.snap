// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header 1`] = `
<Table
  allRowsSelected={false}
  fixedFooter={true}
  fixedHeader={true}
  height="inherit"
  multiSelectable={false}
  selectable={true}
>
  <TableHeader
    adjustForCheckbox={false}
    displaySelectAll={false}
    enableSelectAll={true}
    selectAllSelected={false}
  >
    <TableRow
      displayBorder={true}
      hoverable={false}
      hovered={false}
      selectable={true}
      striped={false}
    >
      <TableHeaderColumn
        className="code"
      >
        Code
      </TableHeaderColumn>
      <TableHeaderColumn
        className="company"
      >
        Company
      </TableHeaderColumn>
      <TableHeaderColumn
        className="price"
      >
        Price
      </TableHeaderColumn>
      <TableHeaderColumn
        className="value"
      >
        Value
      </TableHeaderColumn>
      <TableHeaderColumn
        className="value"
      >
        Change
      </TableHeaderColumn>
      <TableHeaderColumn
        className="value"
      >
        %Change
      </TableHeaderColumn>
    </TableRow>
  </TableHeader>
</Table>
`;
